lib/Steps/Integrations/SvelteKit.ts

```diff
@@ -408,6 +341,112 @@ export class SvelteKit extends BaseIntegration {
     );
   }
 
+  private _wrapHandleError(mod: ProxifiedModule<any>): void {
+    const modAst = mod.exports.$ast as Program;
+    const namedExports = modAst.body.filter(
+      (node) => node.type === 'ExportNamedDeclaration',
+    ) as ExportNamedDeclaration[];
+
+    let foundHandleError = false;
+
+    namedExports.forEach((modExport) => {
+      const declaration = modExport.declaration;
+      if (!declaration) {
+        return;
+      }
+      if (declaration.type === 'FunctionDeclaration') {
+        if (!declaration.id || declaration.id.name !== 'handleError') {
+          return;
+        }
+        foundHandleError = true;
+        yellow(
+          'Cannot safely wrap an `export function handleError` declaration. Please wrap it manually with `handleErrorWithSentry`',
```

Comment:
Why can't we wrap an export function handleError declaration?

Could we make the transform something like this?

```js
export const handleError = Sentry.handleErrorWithSentry(function _handleError() {
  // ...
});
```

Little more complicated for `Sentry.sentryHandle` because of sequence, so I'm fine to log out a warning there.

Reply:
We can, but I'm not sure if there are more problems involved if we convert a function export to an arrow function export. For instance the differences with `this` in the arrow vs normal functions?

Happy to take another look at this. Overall, I just want to avoid doing even more AST modification than we're currently doing. But your proposed transformation looks more reasonable to me than what I came up with before for this case😅 

As for handle, I'm just always adding `sequence`, regardless of if we're the only handler or if there are others. So I think the complexity shouldn't be much higher than for handleError

Reply:
I think if we keep the function declaration (with `function _handleError`) we should be fine - if this takes too much time though (more the 2 hours), it's probably not worth it.